components/ModDashboardLeftNavbar.vue

```diff
@@ -65,6 +66,11 @@ const updateSubmissionListViewState = (submissionListViewOptionValue: SelectedSu
 const updateSelectedDashboardView = (event: Event) => {
     const selectedValue = (event.target as HTMLSelectElement).value
     selectedDashboardView.value = selectedValue
+    if (selectedDashboardView.value === 'facilities') {
+        moderationSubmissionsStore.getSubmissions()
+    } else if (selectedDashboardView.value === 'healthcareProfessionals') {
+        moderationSubmissionsStore.getProfessionals()
+    }
```

Comment:
This part would be perfect if we need to fetch different data. But in this case the healthcare professionals submissions are existing in the `getSubmissions()` so we just need to display different data from that submission query. 

If you go to the `modSubmissionsStore` in the `getSubmissionsGqlQuery` you will see the `healthcareProfessionals` field this is what we want to display if we toggle Healthcare Professionals. At the moment that is still empty so I would use fake data for now as you did.

Reply:
@Diego-Bittencourt I have added seeding data for this to make it easier, so you should be able to use that data instead of creating your own fake data! So the facility and healthcareProfessional field in submissions is now seeded and not empty anymore.